lib/vauth/ntlm.c

```diff
@@ -599,7 +600,7 @@ CURLcode Curl_auth_create_ntlm_type3_message(struct Curl_easy *data,
 
 #if defined(USE_NTRESPONSES) && defined(USE_NTLM2SESSION)
   /* We don't support NTLM2 if we don't have USE_NTRESPONSES */
-  if(ntlm->flags & NTLMFLAG_NEGOTIATE_NTLM2_KEY) {
+  if(ntlm->flags & NTLMFLAG_NEGOTIATE_NTLM_KEY) {
```

Comment:
I don't understand this change. Does this mean that an NTLMv1 message is sent to servers that support NTLMv2 ?

Reply:
That is the part of the code I am not sure about.  The comment is about NTLM2, but the previous section is really the NTLMv2 section i.e. where the correct response is created.  This one seem to create only the LM response.  ( At least the was the result from the wireshark captures) 

Markus